src/__swaps__/screens/Swap/hooks/useAnimatedSwapStyles.ts

```diff
@@ -206,8 +207,12 @@ export function useAnimatedSwapStyles({
   });
 
   const searchOutputAssetButtonStyle = useAnimatedStyle(() => {
+    const isPasteMode = outputProgress.value === NavigationSteps.TOKEN_LIST_FOCUSED && !internalSelectedOutputAsset.value;
+
+    const color = isPasteMode ? foregroundColors['action (Deprecated)'] : internalSelectedOutputAsset.value?.highContrastColor;
```

Comment:
how can I grab the account accent color?

Reply:
I think you can just use blue for the paste button - this blue is a little better than the current blue since this'll respond to light/dark mode:

```
const blue = useForegroundColor('blue');
```

Can get the account color like this though for the future:

```
const { accentColor } = useAccountAccentColor();
```